backend/test/test_language.ml

```diff
@@ -290,10 +294,18 @@ let t_incomplete_propagation () =
     "Incompletes stripped from lists"
     (DList [Dval.dint 5; Dval.dint 6])
     (exec_ast (list [int 5; int 6; fn "List::head" [blank ()]])) ;
+  check_dval
+    "Incompletes stripped from tuples"
```

Comment:
@pbiggar do you agree with this test case?

Reply:
i think probably?